ci-scripts/check_pr_modified_files_for_oai_pipeline.py

```diff
@@ -20,6 +20,9 @@
 mme_is_impacted = False
 
 for line in list_files.split('\n'):
+    res = re.search('lte/gateway/c/core/BUILD.bazel', line)
```

Comment:
There is also a folder `lte/gateway/c/core/common` that seems to have parts needed in the mme build - and I do not see it covered here.
-> might all in `lte/gateway/c/core` be relevant?
-> should `res = re.search('lte/gateway/c/core/oai', line)` (here in line 29) just be changed to `res = re.search('lte/gateway/c/core', line)`? Imho this would then also cover the `BUILD.bazel` file